website/docs/proxy/index.md

```diff
@@ -6,12 +6,365 @@ tags: [podman-desktop, installing, windows, macos, linux]
 keywords: [podman desktop, containers, podman, installing, installation, windows, macos, linux]
 ---
 
-# Using Podman Desktop behind a proxy
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-You can use Podman Desktop behind a proxy in various situations, such as:
+# Installing and using Podman Desktop and Podman in a restricted environment
 
-- [Using a proxy on Windows and macOS](/docs/proxy/using-a-proxy)
-- [Using a proxy requiring a custom Certificate Authorities on macOS and Windows](/docs/proxy/using-a-proxy-requiring-a-custom-ca)
-- [Using a proxy on Linux](/docs/proxy/using-a-proxy-on-linux)
-- [Using a proxy in your containers](/docs/proxy/using-a-proxy-in-your-containers)
-- [Accessing resources behind a VPN with Podman on Windows](/docs/proxy/using-a-vpn-on-windows)
+In a restricted environment you might face this challenges:
+
+- The default Podman Desktop and Podman installation methods download assets during the setup.
+
+  However, a network restricted environment might refuse access to these external resources.
+
+  Consider rather using a specific installation method.
+
+- The Podman Machine receives a network address distinct to your computer network address.
+
+  However, resources in your VPN might be available only for the traffic going through your computer network address.
+
+  Consider enabling the **User mode networking** option when creating your Podman Machine to route the network traffic through your VPN.
+
+- The Podman Machine connects directly to the external network.
+
+  However, a restricted environment might block all traffic no going to a proxy.
+
+  Consider configuring Podman Desktop and your Podman Machine to route the traffic through a proxy.
+
+This tutorial is guiding you through the required steps to work in a restricted environment.
+
+## Installing Podman Desktop and Podman
+
+<Tabs groupId="operating-systems">
+<TabItem value="win" label="Windows">
+
+On Windows, to install Podman Desktop and Podman in a restricted environment, consider using the installer for restricted environments.
+
+- It has all artifacts required to install Podman Desktop and Podman.
+- It does not require Internet access to download resources during installation.
+- However, it does not contain additional utilities, such as Compose or Kind.
```

Comment:
Honestly I feel like this list as well as the sentence above could be heavily minimized to just...

"Follow the following steps if you have no external internet access and you are installing within a restricted environment?"